packages/core/src/config-manager/admin/routes/GetModules.route.ts

```diff
@@ -11,6 +11,7 @@ export function getModulesRoute(registeredModules: Map<string, RegisteredModule>
     {
       path: '/config/modules',
       action: ConduitRouteActions.GET,
+      description: `Returns configuration of all registered modules.`,
```

Comment:
This actually returns an array of module objects, containing their name, url and serving status.
You could phrase this like:
`Returns as array of currently available modules, providing information about their service`